src/config.ts

```diff
@@ -295,6 +299,24 @@ export function semicolonSeparatedList(value: string | undefined): string[] | un
   return value.split(';').map(v => v.trim());
 }
 
+export function parseJsonObject(value: string | undefined): Record<string, string> | undefined {
+  if (!value)
+    return undefined;
+  try {
```

Comment:
I think this code could be simplified a lot, did you mean to parse JSON first and then assert the type of the parsed value?